src/import/loadConfigurationFromIgResource.ts

```diff
@@ -69,7 +70,8 @@ export function loadConfigurationFromIgResource(igRoot: string): Configuration |
       parameters: igResource.definition?.parameter ?? [],
       FSHOnly: true
     };
-    config.dependencies?.forEach(dep => {
+    if (igResource.id) config.id = igResource.id;
```

Comment:
This is a little nitpicky, but... I think most IGs will have an `id` and it looks a little weird to me coming last in the log (because we almost always put `id` first):
```
info  Extracted configuration:
info    canonical: "http://example.org/fshonly"
info    url: "http://example.org/fshonly/ImplementationGuide/tricked-you-because-you-thought-it-was-fhir.example.fshonly"
info    name: undefined
info    packageId: "fhir.example.fshonly"
info    version: "0.1.0"
info    fhirVersion[0]: "4.0.1"
info    dependencies: undefined
info    parameters[0]: {"code":"copyrightyear","value":"2022+"}
info    parameters[1]: {"code":"releaselabel","value":"ci-build"}
info    FSHOnly: true
info    id: "fhir.example.fshonly"
```

What do you think about putting:
```
id: igResource.id,
```
as the _first property_ in the original config object and then in this spot (after the config object) deleting it if it is null/undefined?
```ts
if (config.id == null) {
  delete config.id;
}
```
(Also, I guess prettier doesn't enforce it, but I think we usually use fully formed `if` statements, even when they're a single statement.  I _think_.)

Reply:
That makes sense! Commit [fd89bcd](https://github.com/FHIR/sushi/pull/1016/commits/fd89bcd24176953ce2b4a340e1d502c57db79ef4) implements this suggestion.